source/administration/command-line-tools.rst

```diff
@@ -1380,6 +1541,120 @@ mattermost version
     .. code-block:: none
 
       mattermost version
+      
+mattermost webhook
+---------------
+
+  Description
+    Commands to manage webhooks.
+
+  Child Commands
+    -  `mattermost webhook create-incoming`_ - Create incoming webhook within specific channel. 
+    -  `mattermost webhook create-outgoing`_ - Create outgoing webhook within specific channel.
+    -  `mattermost webhook list`_ - List all webhooks.
+    -  `mattermost webhook modify-incoming`_ - Modify existing incoming webhook by changing its title, description, channel or icon url.
+
+mattermost webhook create-incoming
+~~~~~~~~~~~~~~~~~~~~~~
+
+  Description
+    Create incoming webhook within specific channel 
+
+  Format
+    .. code-block:: none
+
+      mattermost webhook create-incoming 
+
+  Examples
+    .. code-block:: none
+
+      sudo ./mattermost webhook create-incoming --channel [channelID] --user [userID] --display-name [display-name] --description [webhookDescription] --lock-to-channel --icon [iconURL]
+
+  Options
+    .. code-block:: none
+
+          --channel string Channel ID
+          --user string User ID
+          --display-name string Incoming webhook display name
+          --description string Incoming webhook description
+          --lock-to-channel boolean (True/False) Lock incoming webhook to channel
+          --icon [iconURL] Icon URL
+
+mattermost webhook create-outgoing
+~~~~~~~~~~~~~~~~~~~~~~
+
+  Description
+    Create outgoing webhook which allows external posting of messages from a specific channel 
+
+  Format
+    .. code-block:: none
+
+      mattermost webhook create-outgoing
+
+  Examples
+    .. code-block:: none
+
+      sudo ./mattermost webhook create-outgoing --team myteam --channel mychannel --user myusername --display-name mywebhook --description "My cool webhook" --trigger-when 1 --trigger-words "build\ntest" --icon http://localhost:8000/my-slash-handler-bot-icon.png --urls http://localhost:8000/my-webhook-handler --content-type "application/json"
+
+  Options
+    .. code-block:: none
+          --team string [REQUIRED] Team name or ID
+          --channel string Channel name or ID
+          --user string [REQUIRED] User username, email, or ID (required)
+          --display-name string [REQUIRED] Outgoing webhook display name
+          --description string Outgoing webhook description
+          --trigger-words string [REQUIRED] Words to trigger webhook (word1\nword2)
+          --trigger-when. Integer When to trigger webhook (either when trigger word is first (enter 1) or when it's anywhere (enter 0))
```

Comment:
This can be done through PR to the implementation (and later, here), but I would expect to receive here a "first" or "anywhere" as parameter instead of 1 or 0, because makes a lot more sense. There we are exposing the implementation without need.

Reply:
We need to decide if we want to change it. And then, if that is what we want to do, create a ticket for it. But I would keep the current behavior in the documentation anyway, and change it later.